src/styles/utils/_animations.scss

```
@@ -0,0 +1,10 @@
+
+@keyframes move {
```

```suggestion
@keyframes circle {
```
or something like that